.env.example

```diff
@@ -32,11 +33,13 @@ PRISMA_GENERATE_DATAPROXY=
 # ***********************************************************************************************************
 
 # - SHARED **************************************************************************************************
+# Set this to http://app.cal.local:3000 if you want to enable organizations, and
+# check variable ORGANIZATIONS_ENABLED at the bottom of this file
 NEXT_PUBLIC_WEBAPP_URL='http://localhost:3000'
 # Change to 'http://localhost:3001' if running the website simultaneously
 NEXT_PUBLIC_WEBSITE_URL='http://localhost:3000'
 NEXT_PUBLIC_CONSOLE_URL='http://localhost:3004'
-NEXT_PUBLIC_EMBED_LIB_URL='http://localhost:3000/embed/embed.js'
+NEXT_PUBLIC_EMBED_LIB_URL='http://app.cal.local:3000/embed/embed.js'
```

Comment:
This should correspond to `NEXT_PUBLIC_WEBAPP_URL` only

Reply:
Reverting to localhost, forgot to do so when introducing the new orgs env var. Would it be possible to use `NEXT_PUBLIC_WEBAPP_URL` from where `NEXT_PUBLIC_EMBED_LIB_URL` is used and have it only point to `/embed/embed.js` which is the location of the lib? so it's not repetitive and completely dependant on `NEXT_PUBLIC_WEBAPP_URL` from the core.